packages/manager/modules/telecom-universe-components/src/editable-service-name/editable-service-name.html

```diff
@@ -1,57 +1,82 @@
 <form
-    class="editable-service-name"
     name="editableServiceNameForm"
-    data-ng-submit="$ctrl.saveServiceName()"
+    data-ng-submit=":: $ctrl.saveServiceName()"
 >
-    <h1>
-        <!-- NOT IN EDITION -->
+    <div
+        data-ng-hide="$ctrl.inEdition"
+        data-ng-class="{ 'd-flex': !$ctrl.inEdition }"
+    >
+        <h1 class="oui-page-header__title" data-ng-bind="$ctrl.title"></h1>
+
         <span data-ng-if="!$ctrl.inEdition">
-            <span
-                data-ng-bind="$ctrl.title"
-                class="editable-service-name-title d-inline-block align-middle"
-            >
-            </span>
             <button
                 type="button"
-                class="btn btn-editable d-inline-block align-middle"
-                data-ng-click="$ctrl.startEdition()"
+                class="btn btn-editable h-100"
+                data-ng-click=":: $ctrl.startEdition()"
                 data-ng-disabled="$ctrl.disabled"
-                title="{{$ctrl.getTitle()}}"
             >
-                <i class="oui-icon oui-icon-pen_line" aria-hidden="true"></i>
+                <span
+                    class="oui-icon oui-icon-pen_line oui-icon_small"
+                    aria-hidden="true"
+                ></span>
+                <span
+                    class="sr-only"
+                    data-translate="{{:: 'tuc_editableServiceName_change' | translate }}"
```

Comment:
Using the `translate` directive is enough.

Please refer to my previous comment regarding snakeCase naming.

```suggestion
                    data-translate="tuc_editableServiceName_change"
```